### ISSUE TITLE
Server Crash.

### ISSUE THREAD

**Issue by mightyjoe781** [ElCeejo/draconis#21]
Server : multicraft-2
Game : minetest_game 5

Steps to reproduce : Spawn a random dragon using the egg.
```
2022-03-03 07:02:14: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'draconis' in callback luaentity_Step(): ...mo/worlds/delta/worldmods/draconis/api/behaviors.lua:447: bad argument #2 to 'random' (interval is empty)
2022-03-03 07:02:14: ERROR[Main]: stack traceback:
2022-03-03 07:02:14: ERROR[Main]:       [C]: in function 'random'
2022-03-03 07:02:14: ERROR[Main]:       ...mo/worlds/delta/worldmods/draconis/api/behaviors.lua:447: in function 'func'
2022-03-03 07:02:14: ERROR[Main]:       .../prismo/worlds/delta/worldmods/creatura/mob_meta.lua:1157: in function '_execute_utilities'
2022-03-03 07:02:14: ERROR[Main]:       .../prismo/worlds/delta/worldmods/creatura/mob_meta.lua:847: in function <.../prismo/worlds/delta/worldmods/creatura/mob_meta.lua:819>
````

**Comment by dragonfly-ai** [comment]
It's an error in how math.random() gets called.
Change the line to:
            local pos2 = self:get_wander_pos_3d(width + 10 * scale, width + 20 * scale, nil, random(-10, -1) * 0.1)
from:
            local pos2 = self:get_wander_pos_3d(width + 10 * scale, width + 20 * scale, nil, random(-1, -10) * 0.1)

**Comment by Pivert** [comment]
Hi,

Still having the bug, despite the line being as proposed in my (recent) release. I had to disable it again.

`2022-05-28 08:54:25: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'draconis' in callback luaentity_Step(): ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: bad argument #2 to 'random' (interval is empty)
2022-05-28 08:54:25: ERROR[Main]: stack traceback:
2022-05-28 08:54:25: ERROR[Main]:       [C]: in function 'random'
2022-05-28 08:54:25: ERROR[Main]:       ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: in function 'func'
2022-05-28 08:54:25: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:1145: in function '_execute_utilities'
2022-05-28 08:54:25: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:833: in function 'func'
2022-05-28 08:54:25: ERROR[Main]:       ...minetest/bin/../builtin/profiler/instrumentation.lua:107: in function 'old_on_step'
2022-05-28 08:54:25: ERROR[Main]:       .../minetest/bin/../mods/monitoring/builtin/on_step.lua:17: in function <.../minetest/bin/../mods/monitoring/builtin/on_step.lua:15>

--
2022-05-28 08:57:15: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'draconis' in callback luaentity_Step(): ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: bad argument #2 to 'random' (interval is empty)
2022-05-28 08:57:15: ERROR[Main]: stack traceback:
2022-05-28 08:57:15: ERROR[Main]:       [C]: in function 'random'
2022-05-28 08:57:15: ERROR[Main]:       ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: in function 'func'
2022-05-28 08:57:15: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:1145: in function '_execute_utilities'
2022-05-28 08:57:15: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:833: in function 'func'
2022-05-28 08:57:15: ERROR[Main]:       ...minetest/bin/../builtin/profiler/instrumentation.lua:107: in function 'old_on_step'
2022-05-28 08:57:15: ERROR[Main]:       .../minetest/bin/../mods/monitoring/builtin/on_step.lua:17: in function <.../minetest/bin/../mods/monitoring/builtin/on_step.lua:15>

--
2022-05-28 09:01:48: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'draconis' in callback luaentity_Step(): ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: bad argument #2 to 'random' (interval is empty)
2022-05-28 09:01:48: ERROR[Main]: stack traceback:
2022-05-28 09:01:48: ERROR[Main]:       [C]: in function 'random'
2022-05-28 09:01:48: ERROR[Main]:       ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: in function 'func'
2022-05-28 09:01:48: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:1145: in function '_execute_utilities'
2022-05-28 09:01:48: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:833: in function 'func'
2022-05-28 09:01:48: ERROR[Main]:       ...minetest/bin/../builtin/profiler/instrumentation.lua:107: in function 'old_on_step'
2022-05-28 09:01:48: ERROR[Main]:       .../minetest/bin/../mods/monitoring/builtin/on_step.lua:17: in function <.../minetest/bin/../mods/monitoring/builtin/on_step.lua:15>

--
2022-05-28 09:06:58: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'draconis' in callback luaentity_Step(): ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: bad argument #2 to 'random' (interval is empty)
2022-05-28 09:06:58: ERROR[Main]: stack traceback:
2022-05-28 09:06:58: ERROR[Main]:       [C]: in function 'random'
2022-05-28 09:06:58: ERROR[Main]:       ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: in function 'func'
2022-05-28 09:06:58: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:1145: in function '_execute_utilities'
2022-05-28 09:06:58: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:833: in function 'func'
2022-05-28 09:06:58: ERROR[Main]:       ...minetest/bin/../builtin/profiler/instrumentation.lua:107: in function 'old_on_step'
2022-05-28 09:06:58: ERROR[Main]:       .../minetest/bin/../mods/monitoring/builtin/on_step.lua:17: in function <.../minetest/bin/../mods/monitoring/builtin/on_step.lua:15>

--
2022-05-28 09:11:57: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'draconis' in callback luaentity_Step(): ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: bad argument #2 to 'random' (interval is empty)
2022-05-28 09:11:57: ERROR[Main]: stack traceback:
2022-05-28 09:11:57: ERROR[Main]:       [C]: in function 'random'
2022-05-28 09:11:57: ERROR[Main]:       ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: in function 'func'
2022-05-28 09:11:57: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:1145: in function '_execute_utilities'
2022-05-28 09:11:57: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:833: in function 'func'
2022-05-28 09:11:57: ERROR[Main]:       ...minetest/bin/../builtin/profiler/instrumentation.lua:107: in function 'old_on_step'
2022-05-28 09:11:57: ERROR[Main]:       .../minetest/bin/../mods/monitoring/builtin/on_step.lua:17: in function <.../minetest/bin/../mods/monitoring/builtin/on_step.lua:15>

--
2022-05-28 09:26:34: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'draconis' in callback luaentity_Step(): ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: bad argument #2 to 'random' (interval is empty)
2022-05-28 09:26:34: ERROR[Main]: stack traceback:
2022-05-28 09:26:34: ERROR[Main]:       [C]: in function 'random'
2022-05-28 09:26:34: ERROR[Main]:       ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: in function 'func'
2022-05-28 09:26:34: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:1145: in function '_execute_utilities'
2022-05-28 09:26:34: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:833: in function 'func'
2022-05-28 09:26:34: ERROR[Main]:       ...minetest/bin/../builtin/profiler/instrumentation.lua:107: in function 'old_on_step'
2022-05-28 09:26:34: ERROR[Main]:       .../minetest/bin/../mods/monitoring/builtin/on_step.lua:17: in function <.../minetest/bin/../mods/monitoring/builtin/on_step.lua:15>

--
2022-05-28 09:28:15: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'draconis' in callback luaentity_Step(): ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: bad argument #2 to 'random' (interval is empty)
2022-05-28 09:28:15: ERROR[Main]: stack traceback:
2022-05-28 09:28:15: ERROR[Main]:       [C]: in function 'random'
2022-05-28 09:28:15: ERROR[Main]:       ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: in function 'func'
2022-05-28 09:28:15: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:1145: in function '_execute_utilities'
2022-05-28 09:28:15: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:833: in function 'func'
2022-05-28 09:28:15: ERROR[Main]:       ...minetest/bin/../builtin/profiler/instrumentation.lua:107: in function 'old_on_step'
2022-05-28 09:28:15: ERROR[Main]:       .../minetest/bin/../mods/monitoring/builtin/on_step.lua:17: in function <.../minetest/bin/../mods/monitoring/builtin/on_step.lua:15>

--
2022-05-28 09:31:20: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'draconis' in callback luaentity_Step(): ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: bad argument #2 to 'random' (interval is empty)
2022-05-28 09:31:20: ERROR[Main]: stack traceback:
2022-05-28 09:31:20: ERROR[Main]:       [C]: in function 'random'
2022-05-28 09:31:20: ERROR[Main]:       ...test/minetest/bin/../mods/draconis/api/behaviors.lua:447: in function 'func'
2022-05-28 09:31:20: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:1145: in function '_execute_utilities'
2022-05-28 09:31:20: ERROR[Main]:       .../minetest/minetest/bin/../mods/creatura/mob_meta.lua:833: in function 'func'
2022-05-28 09:31:20: ERROR[Main]:       ...minetest/bin/../builtin/profiler/instrumentation.lua:107: in function 'old_on_step'
2022-05-28 09:31:20: ERROR[Main]:       .../minetest/bin/../mods/monitoring/builtin/on_step.lua:17: in function <.../minetest/bin/../mods/monitoring/builtin/on_step.lua:15>
`

**Comment by mightyjoe781** [comment]
Has been patched in 2.0 release
